app.js

```diff
@@ -0,0 +1,450 @@
+        } else {
+            this.sortPeopleListByDate();
+        }
+    },
+    
+    sortPeopleListByName() {
+        const list = app.listOfPeople;
+        list.sort((a, b) => { 
+            if (b.name > a.name) { 
+                return -1;
+            }
+            else { 
+                return 1;
+            }
+        });
+        
+        this.displayPeople(list);
+    },
+    
+    sortPeopleListByDate() {
+        const list = app.listOfPeople;
+        list.sort((a, b) => { 
+            if (b.startDate > a.startDate) {  
+                return -1;
+            } else {
+                return 1;
+            }
+        });
+        
+        this.displayPeople(list);
+    },
+    
+    sortPeopleListByGroup() {
+        const list = app.listOfPeople;
+        list.sort(a => { 
+            if (a.group === 'beginner') {
+                return -1;
+            } else if (a.group === 'intermediate') {
+                return 1;
+            } else  if (a.group === 'advanced') {
+                return 2;
+            }
+        });
+        
+        this.displayPeople(list);
+    },
+    
+    displayPeople(sortedList) {
+        const table = document.querySelector('.display-people-list');
+        table.innerHTML = '';
+        const list = sortedList || app.listOfPeople;
+        list.forEach(person => {
+            const tr = this.createTrElement();
+            for (const prop in person) {
+                const td = this.createTdElement(person[prop]);
+                tr.appendChild(td);
+            }
+            table.appendChild(tr);
+        });
+    },
+    
+    createTrElement() {
+        const tr = document.createElement('tr');
+        return tr;
+    },
+    
+    createTdElement(person, className) {
+        const td = document.createElement('td');
+        td.textContent = person;
+        td.classList.add(className);
+        return td;
+    },
+    
+    createNewMember() {
+        const name = document.querySelector('.member-name-input').value;
+        document.querySelector('.member-name-input').value = '';
+        const startDate = document.querySelector('#start-date').value;
+        document.querySelector('#start-date').value = '';
+        const recommendedBy = document.querySelector('#recommended-input').value || document.querySelector('#list-of-members').value;
+        document.querySelector('#list-of-members').value = '';
+        document.querySelector('#recommended-input').value = '';
+        const group = document.querySelector('#group').value;
+        document.querySelector('#group').value = '';
+        function Member(name, recommendedBy, startDate, group) {
+            this.name = name;
+            this.recommendedBy = recommendedBy;
+            this.startDate = startDate;
+            this.group = group;
+        }
+        app.listOfPeople.push(new Member(name, recommendedBy, startDate, group));
+        appStorage.setListOfPeople();
+    },
+    
+    createEvents() {
```

Comment:
createEvents is misleading for two reasons:
1. it doesn't create events, it binds them
2. it binds a single event